syntax = "proto3";

service MessageTransfer {
  rpc image_processor (MessageRequest) returns (MessageReply) {}
}

message MessageRequest {
  string frame = 1;
  string frame_shape = 2;
}

message MessageReply {
    string result = 1;
    string frame_shape = 2;
}
